From 084a8610269da8b092832465a4b39a4249b5c083 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Y=C7=94ch=C4=93n=20Gu=C5=8D=20=E9=83=AD=E5=AE=87=E7=90=9B?=
 <yguo@posteo.net>
Date: Fri, 17 Nov 2023 01:29:04 +0100
Subject: [PATCH] patch
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Yǔchēn Guō 郭宇琛 <yguo@posteo.net>
---
 zathura/seccomp-filters.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/zathura/seccomp-filters.c b/zathura/seccomp-filters.c
index 61e1a26..598943a 100644
--- a/zathura/seccomp-filters.c
+++ b/zathura/seccomp-filters.c
@@ -214,6 +214,9 @@ seccomp_enable_strict_filter(zathura_t* zathura)
   ALLOW_RULE(recvmsg);
   /*  ALLOW_RULE(restart_syscall); unused? */
   ALLOW_RULE(rseq);
+  ALLOW_RULE(restart_syscall);
+  ALLOW_RULE(clock_gettime);
+  ALLOW_RULE(gettimeofday);
   ALLOW_RULE(rt_sigaction);
   ALLOW_RULE(rt_sigprocmask);
   ALLOW_RULE(sched_setattr);
-- 
2.42.0

